CHANGELOG.md

```diff
@@ -23,6 +23,14 @@ Version template:
 # Alfred Telemetry Changelog
 ## [0.6.1] - UNRELEASED
```

Comment:
Given the amount of changes / fixes / removal, 0.7.0 might be better as next release version?